### DISCUSSION TITLE
ScriptReference/BuildOptions

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#699] [Notes]
# ScriptReference/BuildOptions

https://docs.unity3d.com/ScriptReference/BuildOptions.html

**Comment by rob5300** [comment]
There appears to be an override to ToString() that makes other properties appear in the resulting string.
e.g. `EnableDeepProfilingSupport.ToString()` becomes `"CompressTextures, StripDebugSymbols, ForceOptimizeScriptCompilation, Il2CPP, EnableDeepProfilingSupport"`
